syft/workers/base.py

```diff
@@ -116,11 +116,11 @@ def __init__(
         self._message_router = {
             CommandMessage: self.execute_command,
             PlanCommandMessage: self.execute_plan_command,
-            ObjectMessage: self.set_obj,
+            ObjectMessage: self.handle_object_msg,
             ObjectRequestMessage: self.respond_to_obj_req,
             ForceObjectDeleteMessage: self.rm_obj,  # FIXME: there is no ObjectDeleteMessage
             IsNoneMessage: self.is_tensor_none,
```

Comment:
Don't you need to modify is_tensor_none to take the message with an object id inside as parameter?